### ISSUE TITLE
Initial Docker run, not working

### ISSUE THREAD

**Issue by kd9lsv** [Maescool/docker-Cloudlog#3]
Hello, I am trying to install cloudlog and I get an error on the website. once I run the run.sh Here are the error messages I receive.  Each time I refresh I get the next error message. After the error messages the webpage gives me a 500 error.
![image](https://user-images.githubusercontent.com/65520146/210204877-a078e25d-f1ec-4614-a49c-82e8523cdfc5.png)
![image](https://user-images.githubusercontent.com/65520146/210204897-8855a4cf-cb0c-4b93-b91c-b16e84efa9cf.png)
![image](https://user-images.githubusercontent.com/65520146/210204912-404f0f85-d184-40f6-b1f4-2352a0032852.png)
Here are the docker compose logs if that helps.

```
Attaching to cloudloga_web_1, cloudloga_app_1, cloudloga_cron_1, cloudloga_redis_1, cloudloga_mariadb_1
[33mcron_1     |[0m Making sure volumed directories have right perms
[33mcron_1     |[0m No config file, creating one based on template
[33mcron_1     |[0m Config file has been created.
[33mcron_1     |[0m No DB config file, creating one based on template
[33mcron_1     |[0m DB Config file has been created.
[33mcron_1     |[0m Wating for mysql to come up
[33mcron_1     |[0m done
[35mredis_1    |[0m 1:C 02 Jan 2023 07:40:51.884 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
[35mredis_1    |[0m 1:C 02 Jan 2023 07:40:51.884 # Redis version=6.2.6, bits=64, commit=00000000, modified=0, pid=1, just started
[35mredis_1    |[0m 1:C 02 Jan 2023 07:40:51.884 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.885 * monotonic clock: POSIX clock_gettime
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.885 * Running mode=standalone, port=6379.
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.885 # Server initialized
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.885 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.886 * Loading RDB produced by version 6.2.6
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.886 * RDB age 14 seconds
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.886 * RDB memory usage when created 0.77 Mb
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.886 # Done loading RDB, keys loaded: 0, keys expired: 0.
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.886 * DB loaded from disk: 0.000 seconds
[35mredis_1    |[0m 1:M 02 Jan 2023 07:40:51.886 * Ready to accept connections
[36mapp_1      |[0m Making sure volumed directories have right perms
[36mapp_1      |[0m No config file, creating one based on template
[36mapp_1      |[0m Config file has been created.
[36mapp_1      |[0m No DB config file, creating one based on template
[36mapp_1      |[0m DB Config file has been created.
[36mapp_1      |[0m Wating for mysql to come up
[36mapp_1      |[0m done
[36mapp_1      |[0m table options not found, installing db tables
[36mapp_1      |[0m ERROR 1050 (42S01) at line 424: Table 'dxcc_entities' already exists
[36mapp_1      |[0m removing install dir, not needed anymore
[36mapp_1      |[0m starting php-fpm
[34mweb_1      |[0m 2023/01/02 07:40:55 [warn] 8#8: *1 using uninitialized "fe_https" variable, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", host: "localhost"
[34mweb_1      |[0m 172.25.0.1 - - [02/Jan/2023:07:40:55 +0000] "GET / HTTP/1.1" 500 912 "-" "Mozilla/5.0 (Windows NT 10.0; rv:108.0) Gecko/20100101 Firefox/108.0"
[34mweb_1      |[0m 2023/01/02 07:42:16 [warn] 8#8: *3 using uninitialized "fe_https" variable, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", host: "localhost"
[34mweb_1      |[0m 172.25.0.1 - - [02/Jan/2023:07:42:16 +0000] "GET / HTTP/1.1" 500 681 "-" "Mozilla/5.0 (Windows NT 10.0; rv:108.0) Gecko/20100101 Firefox/108.0"
[34mweb_1      |[0m 2023/01/02 07:42:29 [warn] 8#8: *3 using uninitialized "fe_https" variable, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", host: "localhost"
[34mweb_1      |[0m 172.25.0.1 - - [02/Jan/2023:07:42:30 +0000] "GET / HTTP/1.1" 500 953 "-" "Mozilla/5.0 (Windows NT 10.0; rv:108.0) Gecko/20100101 Firefox/108.0"
[34mweb_1      |[0m 2023/01/02 07:42:42 [warn] 8#8: *3 using uninitialized "fe_https" variable, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", host: "localhost"
[34mweb_1      |[0m 172.25.0.1 - - [02/Jan/2023:07:42:43 +0000] "GET / HTTP/1.1" 500 5 "-" "Mozilla/5.0 (Windows NT 10.0; rv:108.0) Gecko/20100101 Firefox/108.0"
[34mweb_1      |[0m 2023/01/02 07:44:39 [warn] 8#8: *7 using uninitialized "fe_https" variable, client: 172.25.0.1, server: , request: "GET / HTTP/1.1", host: "localhost"
[34mweb_1      |[0m 172.25.0.1 - - [02/Jan/2023:07:44:39 +0000] "GET / HTTP/1.1" 500 5 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"
[32mmariadb_1  |[0m 2023-01-02 07:40:51+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.4.27+maria~ubu2004 started.
[32mmariadb_1  |[0m 2023-01-02 07:40:52+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
[32mmariadb_1  |[0m 2023-01-02 07:40:52+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.4.27+maria~ubu2004 started.
[32mmariadb_1  |[0m 2023-01-02 07:40:52+00:00 [Note] [Entrypoint]: MariaDB upgrade not required
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] mysqld (mysqld 10.4.27-MariaDB-1:10.4.27+maria~ubu2004) starting as process 1 ...
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Using Linux native AIO
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Uses event mutexes
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Number of pools: 1
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Using SSE2 crc32 instructions
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] mysqld: O_TMPFILE is not supported on /tmp (disabling future attempts)
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Initializing buffer pool, total size = 256M, instances = 1, chunk size = 128M
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Completed initialization of buffer pool
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: 128 out of 128 rollback segments are active.
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Creating shared tablespace for temporary tables
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Waiting for purge to start
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: 10.4.27 started; log sequence number 296425; transaction id 842
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] Plugin 'FEEDBACK' is disabled.
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] Server socket created on IP: '::'.
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] InnoDB: Buffer pool(s) load completed at 230102  7:40:52
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] Reading of all Master_info entries succeeded
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] Added new Master_info '' to hash table
[32mmariadb_1  |[0m 2023-01-02  7:40:52 0 [Note] mysqld: ready for connections.
[32mmariadb_1  |[0m Version: '10.4.27-MariaDB-1:10.4.27+maria~ubu2004'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
```

**Comment by Maescool** [comment]
Hi thanks for the report.

I have implemented a fix in commit [55e8bb4](https://github.com/Maescool/docker-Cloudlog/commit/55e8bb4fd62f2278a3fc4010db2717b565a822b1)

you can now docker-compose stop, 
then remove and recreate the storage dir (since it's corrupted anyway),
git pull and run.sh again.

**Comment by kd9lsv** [comment]
thanks for the fix. it works completely as intended now.
you can close this issue.
